setup.py

```diff
@@ -1,28 +1,23 @@
+import json
+import os
 import sys
 
 try:
     from setuptools import setup
 except ImportError as ex:
     from distutils.core import setup
 
-try:
-    from odml.info import AUTHOR, CONTACT, CLASSIFIERS, HOMEPAGE, VERSION
-except ImportError as ex:
-    # Read the information from odml.info.py if package dependencies
-    # are not yet available during a local install.
-    CLASSIFIERS = ""
-    with open('odml/info.py') as f:
-        for line in f:
-            curr_args = line.split(" = ")
-            if len(curr_args) == 2:
-                if curr_args[0] == "AUTHOR":
-                    AUTHOR = curr_args[1].replace('\'', '').replace('\\', '').strip()
-                elif curr_args[0] == "CONTACT":
-                    CONTACT = curr_args[1].replace('\'', '').strip()
-                elif curr_args[0] == "HOMEPAGE":
-                    HOMEPAGE = curr_args[1].replace('\'', '').strip()
-                elif curr_args[0] == "VERSION":
-                    VERSION = curr_args[1].replace('\'', '').strip()
+with open(os.path.join("odml/info.json")) as infofile:
```

Comment:
Typo here?
`os.path.join("odml", "info.json")`
or I guess you could remove the `os.path.join` (unless it's required for Windows)

Reply:
just sloppy, thx for the comment.